conf/app.routes

```diff
@@ -12,27 +12,31 @@ GET         /choice                                controllers.ChoiceController.
 GET         /choice/:journey                       controllers.ChoiceController.startSpecificJourney(journey: forms.Choice)
 POST        /choice                                controllers.ChoiceController.submit()
 
+#Manage MUCR
+GET         /manage-mucr                          controllers.consolidations.ManageMucrController.displayPage()
+POST        /manage-mucr                          controllers.consolidations.ManageMucrController.submit()
```

Comment:
Why we have a separate endpoints for manage MUCR? What is the reason of this page?

Just asking, because not sure if it's related with this blocking the ucr field. Probably I don't understand what is happening and what is the reason of this Controller and endpoints

Reply:
For details, please have a look at the ticket or the prototype.
In short: this page needs to be present if user queried for MUCR in order to determine queried MUCR as parent or child MUCR. Depending on the answer the user is redirected to different page.